h2o-py/h2o/frame.py

```diff
@@ -414,16 +413,16 @@ def show(self, use_pandas=False):
         if self._ex is None:
             print("This H2OFrame has been removed.")
             return
-        if not self._ex._cache.is_valid(): self._frame()._ex._cache.fill()
+        if not self._ex._cache.is_valid(): self._frame()._ex._cache.fill(rows=self.nrows)
```

Comment:
@jakubhava Yes, this behavior is seen in ipython and python terminal, but not in a Jupyter notebook.